Content.Server/Cloning/CloningConsoleSystem.cs

```diff
@@ -148,49 +151,52 @@ public void UpdateUserInterface(EntityUid consoleUid, CloningConsoleComponent co
             _uiSystem.SetUiState(ui, newState);
         }
 
-        public void TryClone(EntityUid uid, EntityUid cloningPodUid, EntityUid scannerUid, CloningPodComponent? cloningPod = null, MedicalScannerComponent? scannerComp = null, CloningConsoleComponent? consoleComponent = null)
+        public void TryClone(EntityUid uid, EntityUid cloningPodUid, EntityUid scannerUid, CloningPodComponent cloningPod, MedicalScannerComponent? scannerComp = null, CloningConsoleComponent? consoleComponent = null)
```

Comment:
Is there a reason `CloningPodComponent` became required for this call?

Reply:
Yes, I swapped CloningPodComponent from being Nullable to Required because it's literally impossible for the entire system to work if the CloningPodComponent doesn't exist(If you delete the CloningPodComponent mid-cloning, it recursively deletes the clone, because the clone's very existence was stored inside the CloningPodComponent). Doing it this way means anytime I go to a subfunction(such as CheckBiomassCost), I no longer need to Resolve the CloningPod, because I already know it exists. Less code tedium that way.

The Console and Scanner cease being required the moment you leave TryClone's guard clause, so they're nullable(We only need to check if they exist exactly once, because this is a public function and might be called by things other than the cloning console).